test/cases/LegacyLenientAttributes.webidl

```diff
@@ -0,0 +1,7 @@
+[Exposed=Window]
+interface LegacyLenientAttributes {
+  [LegacyLenientSetter] readonly attribute DOMString lenientSetter;
+  [LegacyLenientSetter, LenientThis] readonly attribute DOMString lenientThisSetter;
+  [LenientThis] readonly attribute DOMString lenientThis;
+  [LenientThis, Replaceable] readonly attribute DOMString replaceableLenientThis;
```

Comment:
Can you also test

- `[LenientThis]` by itself, non-readonly attribute
- `[Replaceable]` by itself, readonly attribute

?